config/config.go

```diff
@@ -228,6 +228,14 @@ type ScrapeConfig struct {
 	Scheme string `yaml:"scheme,omitempty"`
 	// The HTTP basic authentication credentials for the targets.
 	BasicAuth *BasicAuth `yaml:"basic_auth,omitempty"`
+	// The bearer token for the targets.
+	BearerToken string `yaml:"bearer_token,omitempty"`
```

Comment:
@jimmidyson just a meaningful message. I think there are a few examples for mutual exclusion for other options.